var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -56,6 +57,15 @@ class Sundials(CMakePackage, CudaPackage, ROCmPackage):
         variant(pkg, default=True,
                 description='Enable %s solver' % pkg)
 
+    # Language standards
+    variant('cstd', default='99',
+            description='C language standard',
+            values=('90', '99', '11', '17'))
```

Comment:
Does this break the HIP build? When building outside of spack, I've run into problems with the C standard flags causing a hipcc error.

Reply:
Looks like this might have been addressed with [CMake 3.21.3](https://cmake.org/cmake/help/latest/release/3.21.html#id3)

Reply:
This only breaks the HIP examples. so I have added a conflict for '+rocm+examples' and 6.0.0. Then, the next sundials release will fix the issue.